keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -85,6 +86,17 @@ class ResourceActuator(
         return@withTracingContext
       }
 
+      val deliveryConfig = deliveryConfigRepository.deliveryConfigFor(resource.id)
+      val environment = checkNotNull(deliveryConfig.environmentFor(resource)) {
+        "Failed to find environment for ${resource.id} in deliveryConfig ${deliveryConfig.name}"
+      }
+
+      if (deliveryConfig.isPromotionCheckStale()) {
+        log.debug("Environment check for {} is {} old, skipping checks", id, deliveryConfig.isPromotionCheckStale())
```

Comment:
```suggestion
        log.debug("Promotion check for environment {} in delivery config {} is old, skipping checks for resource {}", environment.name, deliveryConfig.name, id)
```

Reply:
Ugh, I think I originally wrote the log line when I was using a `StopWatch` to time the execution. I'll fix this up